Sources/SmokeHTTPClient/HTTPOperationsClient.swift

```diff
@@ -100,6 +100,50 @@ public struct HTTPOperationsClient {
                                             configuration: clientConfiguration)
     }
     
+    /**
+     Initializer.
+
+     - Parameters:
+         - endpointHostName: The server hostname to contact for requests from this client.
+         - endpointPort: The server port to connect to.
+         - contentType: The content type of the payload being sent by this client.
+         - clientDelegate: Delegate for the HTTP client that provides client-specific logic for handling HTTP requests.
+         - timeoutConfiguration: The timeout configuration to use
+         - eventLoopProvider: Provides the event loop to be used by the client.
+                              If not specified, the client will create a new multi-threaded event loop
+                              with the number of threads specified by `System.coreCount`.
+         - optional configuration for the connection pool. If not provided, the default configuration is used.
+     */
+    public init(endpointHostName: String,
```

Comment:
Can we make the previous constructor (at line 71) call this constructor to reduce code deuplication?

Reply:
Good point, done.